pulp_rpm/app/serializers.py

```diff
@@ -331,85 +343,189 @@ class Meta:
+        help_text=_("Date when the update was issued (e.g. '2013-12-02 00:00:00')"),
+        required=False
     )
     fromstr = serializers.CharField(
         help_text=_("Source of the update (e.g. security@redhat.com)"),
         allow_blank=True
     )
     status = serializers.CharField(
         help_text=_("Update status ('final', ...)"),
-        allow_blank=True
+        allow_blank=True,
+        required=False
     )
     title = serializers.CharField(
         help_text=_("Update name"),
-        allow_blank=True
+        allow_blank=True,
+        required=False
     )
     summary = serializers.CharField(
         help_text=_("Short summary"),
-        allow_blank=True
+        allow_blank=True,
+        required=False
     )
     version = serializers.CharField(
         help_text=_("Update version (probably always an integer number)"),
-        allow_blank=True
+        allow_blank=True,
+        required=False
     )
 
     type = serializers.CharField(
         help_text=_("Update type ('enhancement', 'bugfix', ...)"),
-        allow_blank=True
+        allow_blank=True,
+        required=False
     )
     severity = serializers.CharField(
         help_text=_("Severity"),
-        allow_blank=True
+        required=False
     )
     solution = serializers.CharField(
         help_text=_("Solution"),
-        allow_blank=True
+        required=False
     )
     release = serializers.CharField(
         help_text=_("Update release"),
-        allow_blank=True
+        allow_blank=True,
+        required=False
     )
     rights = serializers.CharField(
         help_text=_("Copyrights"),
-        allow_blank=True
+        required=False
     )
     pushcount = serializers.CharField(
         help_text=_("Push count"),
-        allow_blank=True
+        required=False
+    )
+    reboot_suggested = serializers.BooleanField(
+        help_text=_("Reboot suggested"),
+        required=False
     )
     pkglist = UpdateCollectionSerializer(
         source='collections', read_only=True,
-        many=True, help_text=_("List of packages")
+        many=True, help_text=_("List of packages"),
+        required=False
     )
     references = UpdateReferenceField(
         source='pk', read_only=True,
-        help_text=_("List of references")
+        help_text=_("List of references"),
+        required=False
     )
 
+    def create(self, validated_data):
+        """
+        Create UpdateRecord and its subclasses from JSON file.
+
+        Returns:
+            UpdateRecord instance
+
+        """
+        references = validated_data.pop("references", [])
+        pkglist = validated_data.pop("pkglist", [])
+        update_collections_to_save = list()
+        update_collection_packages_to_save = list()
+        update_references_to_save = list()
+        try:
+            update_record = super().create(validated_data)
+        except IntegrityError:
```

Comment:
what will happen in case `repository` was provided in the call?

Reply:
new repository version is created
```
"created_resources": [
        "/pulp/api/v3/repositories/rpm/rpm/229edbce-23a8-47c1-8ec0-6ac6fd7e77f1/versions/1/",
        "/pulp/api/v3/content/rpm/advisories/145bf310-3bf8-47e2-a1cc-cc38e847f17e/"
    ],
```